www/_blog/2021-10-04-supabase-beta-sept-2021.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Supabase Beta Sept 2021
+description: Hackathon, Aborting request, UI updates, and now Hiring.
+author: paul_copplestone
+author_url: https://github.com/kiwicopple
+author_image_url: https://github.com/kiwicopple.png
+authorURL: https://github.com/kiwicopple
+image: 2021-sept/release-sept-2021.jpg
+thumb: 2021-sept/release-sept-2021-cover.jpg
+tags:
+  - release-notes
+date: '2021-10-04'
+toc_depth: 3
+---
+
+Did you know it's been 2 years since the [first commit](https://github.com/supabase/realtime/commit/175f649784147af80acfc9ff5be9d160285c76ea) to Realtime, our real-time engine for Postgres? Before we even existed as a company! 
+We spent this month improving docs and content content, improving UX, and [onboarding Developer Advocates](https://twitter.com/thorwebdev/status/1441041268411277322)!
+
+## Hackathon v2
+
+To kick off [Hacktoberfest](https://hacktoberfest.digitalocean.com/), another Supabase Hackathon is happening [right now](https://supabase.io/blog/2021/09/28/supabase-hacktoberfest-hackathon-2021). You've got another 7 days to be in to win a limited edition t-shirt. 
+
+![Hackathon](/new/images/blog/2021-sept/hacktober.png)
+
+## Abort Requests
+
+We added support for [AbortController](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) in our Javascript library so that you can abort long-running queries. [[Docs](https://supabase.io/docs/reference/javascript/select#aborting-requests-in-flight)]
+
+![Abort Requests](/new/images/blog/2021-sept/Supabase_abort_requests.png)
+
+## Improved table management
+
+We've made a number of changes to the Dashboard to expose some great features of PostgreSQL including:
+
+### Column types
+
+We've improved the column Type field so that it supports your [custom types](https://www.postgresql.org/docs/current/sql-createtype.html).
+
+![Column types](/new/images/blog/2021-sept/columns.png)
+
+### Is Unique
+
+We've made it simple to add a unique constraint.
+
+![Is Unique](/new/images/blog/2021-sept/is-unique.png)
+
+### Edit columns
```

Comment:
Should we mention that it's both adding (when creating new tables) and editing (for existing tables) in bulk? or is the latter sufficient actually